commons/connector/health/health_appleExerciseTime.avsc

```diff
@@ -0,0 +1,24 @@
+{
+  "namespace": "org.radarcns.connector.health",
+  "name": "HKQuantityTypeIdentifierAppleExerciseTime",
+  "type": "record",
+  "doc": "AppleExerciseTime , unit is min.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Apple Healthkit ."
+    },
+    {
+      "name": "timeInterval",
+      "type": "int",
+      "doc": "Chronological window size (s)."
+    },
+    {
+      "name": "appleExerciseTime",
```

Comment:
The package identifies that this is about Apple HealthKit. This can be renamed to `exerciseDuration` to avoid confusion with a timestamp. In the field doc, you could document what Apple HealthKit variable this exerciseDuration corresponds to, e.g. HKQuantityTypeIdentifierAppleExerciseTime.